sdk/nodejs/types/output.ts

```diff
@@ -33842,7 +33842,7 @@ export namespace iot {
         topic: string;
     }
 
-    export interface TopicRuleKinesis {
+    export interface TopicRuleKinese {
```

Comment:
Related - very weird/wrong depluralization change here and in a few other places.

Reply:
Fixed by https://github.com/pulumi/pulumi-aws/pull/2615/commits/743dae9afd4bc455eb0319ca0d26114427fa9e89.